File: android/src/main/java/com/github/DudasMiklos/flutter_fullpdfview/FlutterFullpdfviewPlugin.java
package android.src.main.java.com.github.DudasMiklos.flutter_fullpdfview;

import io.flutter.plugin.common.PluginRegistry.Registrar;

public class FlutterFullpdfviewPlugin {
    /** Plugin registration. */
    public static void registerWith(Registrar registrar) {
        registrar
                .platformViewRegistry()
                .registerViewFactory(
                        "plugins.DudasMiklos.io/pdfview", new FULLPDFViewFactory(registrar.messenger()));
    }
}

File: ios/Classes/FlutterFullpdfviewPlugin.m
#import "FlutterFullpdfviewPlugin.h"
#import "FlutterFullPDFView.h"

@implementation FlutterFullpdfviewPlugin
+ (void)registerWithRegistrar:(NSObject<FlutterPluginRegistrar>*)registrar {
    FLTPDFViewFactory* pdfViewFactory = [[FLTPDFViewFactory alloc] initWithMessenger:registrar.messenger];
    [registrar registerViewFactory:pdfViewFactory withId:@"plugins.DudasMiklos.io/pdfview"];
}
@end
